src/vue/backend/src/store/modules/agentSettings/actions.js

```diff
@@ -1,2 +1,8 @@
+const enableAgentSettings = ({ commit }, payload) => {
```

Comment:
Enabled agent settings is not exactly what this is doing, instead it in enabling the agents plugin. Try `enableImpressAgents`